api_specifications/case-api-0.1.0.yaml

```
@@ -512,7 +512,33 @@ paths:
+          '200':
+            description: ResponseInfo with witness details added to case successfully
+            content:
+              '*/*':
+                schema:
+                  $ref: '#/components/schemas/WitnessDetailsResponse'
+          '400':
+            description: Adding witness to a case failed
+            content:
+              '*/*':
+                schema:
+                  $ref: >- 
```

YAMLlint (1.37.1)

[error] trailing spaces

(trailing-spaces)